.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
 - pipenv run python network-api/manage.py collectstatic --no-input
 before_script:
 - psql -c 'create database network;' -U postgres
+- pipenv run python network-api/manage.py migrate --no-input
```

Comment:
It's not necessary: migrations will be applied when the test database is created (run it with verbose level 2 if you're curious)

Reply:
And that's what I thought, but my test runs on travis were failing because none of the tables were getting created. I'll remove this line and cross my fingers that the issues I saw don't return